test/diagnostics/Localization/fr_localization.swift

```diff
@@ -1,8 +1,9 @@
 // RUN: %empty-directory(%t)
 // RUN: swift-serialize-diagnostics --input-file-path=%S/Inputs/fr.yaml --output-directory=%t/
-// RUN: swift-serialize-diagnostics --input-file-path=%S/Inputs/en.yaml --output-directory=%t/
+// RUN: swift-serialize-diagnostics --input-file-path=%S/Inputs/en.yaml --output-directory=%t/ 2>&1 | %FileCheck %s
 // RUN: %target-typecheck-verify-swift -localization-path %t -locale fr
 
+// CHECK: These diagnostic IDs are no longer availiable: 'not_available_in_def'
```

Comment:
It would be great to add a test which has multiple unknown ids and they are placed intermixed with other diagnostics in the yaml file to make sure parser behaves correctly in such scenarios.